LibsNoSamples.sln

```diff
@@ -1,136 +1,173 @@
-		{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82}.Release|Any CPU.Build.0 = Release|Any CPU
-		{E7F9B699-4941-4053-AB81-7E26C4D24805}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{E7F9B699-4941-4053-AB81-7E26C4D24805}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{E7F9B699-4941-4053-AB81-7E26C4D24805}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{E7F9B699-4941-4053-AB81-7E26C4D24805}.Release|Any CPU.Build.0 = Release|Any CPU
-		{3DC6EC76-D350-4D43-B206-A4CEC8AA36D4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{3DC6EC76-D350-4D43-B206-A4CEC8AA36D4}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{3DC6EC76-D350-4D43-B206-A4CEC8AA36D4}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{3DC6EC76-D350-4D43-B206-A4CEC8AA36D4}.Release|Any CPU.Build.0 = Release|Any CPU
-		{60117A9B-4BB8-472E-BFFF-52CBF67CA95A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{60117A9B-4BB8-472E-BFFF-52CBF67CA95A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{60117A9B-4BB8-472E-BFFF-52CBF67CA95A}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{60117A9B-4BB8-472E-BFFF-52CBF67CA95A}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{AF0E7BEC-90A3-46B4-AEDC-4162D7873AE8} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{DA8E7933-0A56-4668-9786-ECECA3D16944} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{30287EC8-BD85-41CC-BDCB-9DBF9EFBCC98} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{E22918A8-8FE6-47D2-B65E-34436918C5D7} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{D3033A40-F2C2-4495-BC9F-C3C6D497D1CC} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{528C5488-C0D8-45C3-BEAC-3E52C22631C9} = {528F34D9-FF6B-4AAE-8367-182EC19DAB5F}
-		{A8C2D397-BA21-4BD2-93AA-801B0E0804BC} = {7930A443-1981-49CD-8AF5-02E25F3A7FF3}
-		{7930A443-1981-49CD-8AF5-02E25F3A7FF3} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{70DFE67C-D2D1-4BE4-B785-F4445C6F652D} = {7930A443-1981-49CD-8AF5-02E25F3A7FF3}
-		{21B0D02E-8E52-4ECD-BFD3-5F14FCB73703} = {7930A443-1981-49CD-8AF5-02E25F3A7FF3}
-		{8672FC71-00DE-491D-BCEB-101A2A7887E6} = {7930A443-1981-49CD-8AF5-02E25F3A7FF3}
-		{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82} = {7930A443-1981-49CD-8AF5-02E25F3A7FF3}
-		{E7F9B699-4941-4053-AB81-7E26C4D24805} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-		{3DC6EC76-D350-4D43-B206-A4CEC8AA36D4} = {5D231979-0A13-456D-B9F9-CED41F935FD9}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {6C88EF77-AF03-4F82-B666-1831DABAA119}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28621.142

```

Comment:
what will happen when I use vs 2017?

Reply:
It works fine.  I think this is just a comment to look for possible migration scenarios.  I've opened in VS2017 and it works.